### PR TITLE
Feature/product screen

### PR DESCRIPTION
**Pull request by jotasenati**

### CODE REVIEW

**Comment by gabrieldev525** [comment]
You need set the destination of branch to master. To do this, click in edit button, and click in dropdown witih name master and change it to develop

**Comment by jotasenati** [comment]
Screenshot_1

File: frontend/src/components/tabela/table.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    AiOutlinePlus
+} from 'react-icons/ai'
+
+import '../../static/css/cadastroProduto.scss'
+
+
+const produtos = [
+    {
+
+        nomeDoProduto: 'Camisa Social Preta',
```
Comment (garciamendes) [inline frontend/src/components/tabela/table.js:20]: takes away the spaces

File: frontend/src/components/tabela/table.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    }
+
+];
+
+const TableHeadbutton = () => {
+    return (
+        <thead>
+            <tr>
+                <th><input className="inputProcurar" placeholder="Buscar por nome de produto"></input> <button className="botaoBuscar">Buscar</button></th>
+                <th><button className="cadastrar" ><AiOutlinePlus color='#fffff' size={12} /> Cadastrar</button> </th>
```
Comment (garciamendes) [inline frontend/src/components/tabela/table.js:56]: fix indentations

File: frontend/src/components/tabela/table.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+//     checkboxes[i].checked = source.checked;
+//     }
+// }   
+
+const TableHead = () => {
+    return (
+        <thead>
+            <tr>
+                <th><input type="checkbox" id="checkTodos" ></input></th>
+                <th>Nome do produto</th>
```
Comment (garciamendes) [inline frontend/src/components/tabela/table.js:76]: same comment from above

File: frontend/src/components/tabela/table.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    render() {
+
+        return (
+            <>
+                <table className="tabela">
+                    <TableHeadbutton />
+                    <TableHead />
+                    <TableBody produtos={produtos} />
+                    <div class="pagination">
+                        <a href="#">&laquo;</a>
```
Comment (garciamendes) [inline frontend/src/components/tabela/table.js:141]: same comment from above

File: frontend/src/static/css/cadastroProduto.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
+#form form input{
```
Comment (garciamendes) [inline frontend/src/static/css/cadastroProduto.scss:1]: you can follow a tree with sass

**Pull request merged**